prime/src/components/borrow/actions/UniswapAddLiquidityActionCard.tsx

```diff
@@ -259,11 +259,6 @@ export default function UniswapAddLiquidityActionCard(props: ActionCardProps) {
     );
   }
 
-  const max0 = accountState.assets.token0Raw;
-  const max1 = accountState.assets.token1Raw;
-  const maxString0 = Math.max(0, max0 - 1e-6).toFixed(6);
-  const maxString1 = Math.max(0, max1 - 1e-6).toFixed(6);
-
```

Comment:
For consistency with other action cards, can we keep the maxes up here? One option would be to do `let max0 = ...` and `let max1 = ...` and then you can simply do `if (!isToken0Selected) [max0, max1] = [max1, max0]` to flip them

Reply:
I did some looking around, and it appears we are relatively inconsistent with this, but for this PR I agree we should stick with what it was. I plan on going through a second time in a separate PR and making these all consistent.